src/commands/new/parachain.rs

```diff
@@ -83,7 +83,7 @@ impl NewParachainCommand {
 				outro_cancel("git signature could not be found. Please configure your git config with your name and email")?;
 			}
 		}
-		spinner.stop("Generation complete");
+		spinner.stop(format!("Generation complete using {}", tag));
```

Comment:
Suggestion: can the `polkadot` version tag be logged as a separate line? 
`Generation Complete` - on spinner stop
then 
`Version: polkadot-v*`

Reply:
Version is now logged clearly on a separate line.